watertap/tools/parameter_sweep/parameter_sweep.py

```diff
@@ -692,167 +701,9 @@ def parameter_sweep(self, *args, **kwargs):
         pass
 
 
-class ParameterSweep(_ParameterSweepBase):
+class ParameterSweep(_ParameterSweepBase, _ParameterSweepParallelUtils):
```

Comment:
What is going on that we need multiple inheritance here?

Is this just to separate the methods or does some other class need to just inherit from `_ParameterSweepBase`?

Reply:
its so both ParameterSweep and DifferentialParamterSweep class can get access to same underlying parallel functions. 

I was thinking of just putting them into _ParamterSweepBase as well, but figured it would be clenaer to store all relevant parallel function in a single place. 

We can also have _ParamterSweepBase inhert _ParameterSweepParallelUtils as ParatemterSweep and DifferntialParamterSweep inherit _ParamterSweepBase.  

Ultimately, the underlying functions would need to move out of paramtersweep class and either into their own class or _ParmaterSweepBase.  We can do what you think is better/cleaner.